packages/sites/src/domains/index.ts

```diff
@@ -1,19 +1,30 @@
-export * from "./_check-status-and-parse-json";
-export * from "./_get-auth-header";
-export * from "./_get-domain-service-url";
-export * from "./_lookup-portal";
-export * from "./add-domain";
-export * from "./domain-exists";
-export * from "./domain-exists-portal";
-export * from "./get-domains-for-site";
-export * from "./get-unique-domain-name";
-export * from "./get-unique-domain-name-portal";
-export * from "./is-domain-for-legacy-site";
-export * from "./is-domain-used-elsewhere";
-export * from "./is-valid-domain";
-export * from "./lookup-domain";
-export * from "./remove-domain";
-export * from "./types";
-export * from "./update-domain";
-export * from "./_ensure-safe-domain-length";
-export * from "./ensure-unique-domain-name";
+export {
+  /* istanbul ignore next */
```

Comment:
Bummer you needed these ignore comments.

Reply:
Yeah, I tried to avoid. You can see that I wrote a test to cover this file, but it's no good.